.gitignore

```diff
@@ -1 +1,2 @@
 node_modules/
+.idea
```

Comment:
No. you should have the .idea folder as an part of your global gitignore. Not part of this repo.